src/schemas/space.json

```diff
@@ -234,8 +234,31 @@
           "required": ["name"],
           "additionalProperties": false
         },
-        "delegation": {
-          "type": "boolean"
+        "delegationPortal": {
+          "standard": {
+            "type": "string",
+            "title": "Standard",
+            "description": "The standard used by your delegation contract",
+            "anyOf": [
+              { "const": "compound-governor", "title": "Compound governor" }
+            ]
+          },
+          "contract": {
+            "type": "string",
+            "format": "address",
+            "title": "Contract address",
+            "description": "The address of your delegation contract",
+            "examples": ["0x3901D0fDe202aF1427216b79f5243f8A022d68cf"]
+          },
+          "subgraphUrl": {
```

Comment:
Depending on the standard the API may not be a subgraph, I would call it `delegationApi` instead.